docs/source/noise/index.rst

```diff
@@ -0,0 +1,58 @@
+.. _noise_main:
+
+======
+ Noise
+======
+
+.. contents::
+   :depth: 2
+   :local:
+   :backlinks: none
+
+In order to have a realistic challenge with entity resolution, it is essential
+to add noise to the data. Pseudopeople can add two broad categories of noise to
+the datasets it generates:
+
+#. **Row-based noise:** Errors in the inclusion or exclusion of entire rows of
+   data, such as duplication or omission
+#. **Column-based noise:** Errors in individual data entry, such as miswriting
```

Comment:
@zmbc How does this sound?

1. **Row-based noise:** Errors in the inclusion or exclusion of entire rows of
   data, such as duplication or omission of a record
1. **Column-based noise:** Errors in data entry for individual fields within a
   record, such as miswriting or incorrectly selecting responses

Reply:
Sounds good to me! Could use "row" instead of "record" for consistency but we'll definitely be using both throughout the docs so no big deal.

Reply:
I used both on purpose in case some people are more familiar with one vs. the other. I edited slightly to make it more consistent, but then added a paragraph underneath where I introduce the term "record."